src/commands/cloud-run/flare.ts

```diff
@@ -141,19 +200,10 @@ export class CloudRunFlareCommand extends Command {
 
       // Confirm before sending
       this.context.stdout.write('\n')
-      let confirmSendFiles
-      try {
-        confirmSendFiles = await requestConfirmation(
-          'Are you sure you want to send the flare file to Datadog Support?',
-          false
-        )
-      } catch (err) {
-        if (err instanceof Error) {
-          this.context.stderr.write(helpersRenderer.renderError(err.message))
-        }
-
-        return 1
-      }
+      const confirmSendFiles = await requestConfirmation(
+        'Are you sure you want to send the flare file to Datadog Support?',
+        false
+      )
```

Comment:
Since this is all wrapped by an outer try/catch, this try/catch here is redundant.